### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by Stretch96**
* The commands available in dalmatian-tools, only work with the dalmatian infrastructures launched from the dxw/dalmatian repo (which is private and for internal use only) - we call this dalmatian v1
* There are also commands which only work with the infrastructures deployed via dalmatian-tools - which we call v2
* This change refactors the structure of the commands available in dalmatian tools. The subcommand directories now contain a v1 and/or v2 directory, where the relevant script can be placed. This allows us to have commands named the same, but can be used on either version
* The version command can be used to show the current version, or to switch between them. When on v1, only v1 commands will be available. Which on v2, only v2 commands will be available. If they are needed on both, a symlink can be created between them

### CODE REVIEW

**Comment by DrizzlyOwl** [comment]
I like this idea a lot :)

**Comment by rjw1** [comment]
Since I care this seems to have really broken tab completion.

File: bin/configure-commands/v2/version [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  log_info -l "launched with the dxw/dalmatian repo, which is private and internal" -q "$QUIET_MODE"
+  log_info -l "To use tooling for use with infrastructures deployed via dalmatian-tools," -q "$QUIET_MODE"
+  log_info -l "switch to 'v2' by running 'dalmatian version -v 2'" -q "$QUIET_MODE"
+fi
+if [ "$VERSION" == "v2" ]
+then
+  log_info -l "The tooling available in v2 is to be used with infrastructures" -q "$QUIET_MODE"
+  log_info -l "deployed via dalmatian-tools" -q "$QUIET_MODE"
+  log_info -l "To use tooling for use with infrastructures launched with the dxw/dalmatian repo," -q "$QUIET_MODE"
+  log_info -l "switch to 'v1' by running 'dalmatian version -v 2'" -q "$QUIET_MODE"
```
Comment (rjw1) [inline bin/configure-commands/v2/version:82]:
```suggestion
  log_info -l "switch to 'v1' by running 'dalmatian version -v 1'" -q "$QUIET_MODE"
```

**Review by rjw1 — CHANGES_REQUESTED** [review]

**Comment by rjw1** [comment]
@Stretch96 im happy with the completion stuff now :)

**Review by rjw1 — APPROVED** [review]

**Pull request merged**